deploy/docker/notebook/Dockerfile.template

```diff
@@ -49,6 +49,21 @@ RUN echo "auth requisite pam_deny.so" >> /etc/pam.d/su && \
     fix-permissions $HOME && \
     fix-permissions "$(dirname $CONDA_DIR)"
 
+# Install Lmod
+RUN mkdir -p /build
+WORKDIR /build
```

Comment:
I know that HPC/Singularity support isn't really in scope but we should probably think about ways to not use workdir.  

That would be an issue for using this dockerfile in HPC.  Maybe we can create a story to refactor this if we want to be able to deploy jupyter on HPC with our environment.

Reply:
I know WORKDIR is the issue for *running* container in HPC, i.e. you set a `WORKDIR=/opt/apps/` and then do `CMD ["./myapp"]`. This way Singularity overwrite/ignores(?) WORKDIR and container cannot run.

My question is if WORKDIR creates any issues for intermediary *build* stages? In this case I need to jump into temporary /build folder to install Lmod.

Also, I see that I use ENTRYPOINT and I think that might also cause some issues for Singularity as well.

If you don't mind, I'm going to ignore these issues for this release (0.9.0) and instead create an issue on this repo and corresponding Jira ticket to research Singularity compatibility of the notebook container and fix all issues with that

Reply:
Opened the issue: https://github.com/LabShare/notebooks-deploy/issues/114

Based on a recent discussion, this is a low-priority issue.